devel/rizin/Portfile

```diff
@@ -1,20 +1,22 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           meson 1.0
+PortGroup           github  1.0
+PortGroup           meson   1.0
 
-github.setup        rizinorg rizin 0.2.1 v
+github.setup        rizinorg rizin 0.3.0 v
 revision            0
 epoch               1
 categories          devel
 platforms           darwin
 license             LGPL-3
-maintainers         {l2dy @l2dy} openmaintainer
+maintainers         {l2dy @l2dy} \
+                    {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
 description         UNIX-like reverse engineering framework and command-line toolset
 long_description    Rizin is a fork of the radare2 reverse engineering framework with a focus on usability, \
                     working features and code cleanliness.
-homepage            https://rizin.re/
+homepage            https://rizin.re
```

Comment:
Why would you remove the last slash? Links copied from the address bar of Firefox and Chrome always have this last slash.

Reply:
Repology used to highlight project links that ended in slash alone, as reported by the Problems section: https://repology.org/repository/macports/problems 

In light of this, I made a habit of cleaning up the ending slash in links.  The slash gets added automatically by the browser either way.

Since it looks like Repology is no longer highlighting this as a problem, I've returned the link to the way it was prior.